src/label_studio_sdk/converter/converter.py

```diff
@@ -282,12 +281,16 @@ def _get_data_keys_and_output_tags(self, output_tags=None):
         for name, info in self._schema.items():
             if output_tags is not None and name not in output_tags:
                 continue
-            data_keys |= set(map(itemgetter("value"), info["inputs"]))
+            for input_tag in info["inputs"]:
+                for value_key_name in ["value", "valueList"]:
```

Comment:
not sure it matters, but seems here we'd allow having both value and valueList, whereas above we wouldn't. Don't know enough about when we see each to know if that would ever happen or matter, but wanted to make sure that's acceptable

Reply:
it's a very good callout! I think we perform label config validation upstream of this, and should probably reject an input tag with both `value` and `valueList` on it at that level